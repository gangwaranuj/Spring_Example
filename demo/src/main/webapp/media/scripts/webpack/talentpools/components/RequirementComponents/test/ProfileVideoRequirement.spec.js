import React from 'react';
import { shallow } from 'enzyme';
import configureStore from '../../../configureStore';
import ProfileVideoRequirement from '../ProfileVideoRequirement';

describe('<ProfileVideoRequirement />', () => {
	const dummyFunc = jest.fn();
	const store = configureStore();

	describe('Rendering consistency', () => {
		it('should render the same way every time with all props', () => {
			const snapshot = shallow(
				<ProfileVideoRequirement.WrappedComponent
					store={ store }
					dispatch={ dummyFunc }
					applyRequirement={ dummyFunc }
				/>,
			);
			expect(snapshot).toMatchSnapshot();
		});
	});
});
