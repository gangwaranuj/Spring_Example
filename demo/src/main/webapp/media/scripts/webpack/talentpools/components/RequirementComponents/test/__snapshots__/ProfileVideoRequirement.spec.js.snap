// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ProfileVideoRequirement /> Rendering consistency should render the same way every time with all props 1`] = `
<WMFormRow
  id="requirement__profilevideo"
>
  <div
    style={
      Object {
        "display": "flex",
        "flexDirection": "row-reverse",
      }
    }
  >
    <WMRaisedButton
      label="Add this requirement"
      onClick={[Function]}
      secondary={true}
      style={
        Object {
          "margin": "1em 0 0 1em",
        }
      }
    />
  </div>
</WMFormRow>
`;
